pelias-config/README.md

```diff
@@ -0,0 +1,56 @@
+# Custom Pelias Instance
+
+This project is configured to download/prepare/build a complete Pelias installation.
+
+The expensive interpolation, polylines, and placeholder containers have been removed. Some Elasticsearch data is shipped to avoid these expensive Pelias components.
+
+The synonyms folder contains Elasticsearch synonyms which can be used to help Elasticsearch find otherwise difficult to search for points. Its initial values are copied from the Pelias example project. These can be used to address any search issues which arise.
+
+A rudimentary example custom CSV file is included in the `data/csv` folder.
+
+# Setup
+
+Please refer to the instructions at https://github.com/pelias/docker in order to install and configure your docker environment.
+
+The minimum configuration required in order to run this project are [installing prerequisites](https://github.com/pelias/docker#prerequisites), [install the pelias command](https://github.com/pelias/docker#installing-the-pelias-helper-script) and [configure the environment](https://github.com/pelias/docker#configure-environment).
+
+Be careful to not run the Pelias default project. The prepare and interpolation steps especially can take over an hour to run!
+
+If Pelias is installed globally, then running the commands listed in the build step below will work, provided all configuration files have been decrypted.
```

Comment:
I'm ok with offloading the setup of docker/pelias to the provided links, but what configuration files does this refer to? I really love to see setup docs that walk you from cloning to up-and-running. E.g., 
```bash
git clone https://github.com/ibi-group/transit-pois-pelias
cp sample.config config # is this a missing step?
pelias compose pull
pelias elastic start
pelias elastic wait
pelias import csv
pelias compose up

```

Reply:
Have expanded this a little bit in bd0dd663a5578f8c305459c775f3a7881844d257 and 802d60ea62f0b00a44a56023e32763259e8fe22f. I'm weary of copying in the Pelias command install instructions in case Pelias makes changes to their README.